controllers/virtualmachine/volumebatch/volumebatch_controller.go

```diff
@@ -158,6 +159,12 @@ func AddToManager(ctx *pkgctx.ControllerManagerContext, mgr manager.Manager) err
 		}
 	}
 
+	// Setup deferred instance storage PVC cache
+	if pkgcfg.FromContext(ctx).Features.InstanceStorage {
+		r.instanceStoragePVCCache = NewDeferredInstanceStoragePVCCache(
```

Comment:
With AllDisksArePVCs/VMSharedDisks we're already watching PVCs in this controller - and we're client.Get() PVCs elsewhere - so there isn't a reason anymore for a separate watcher for just the instance storage PCVs.

I don't think we are but if we're using that reader to do a List() then when using the regular client we should include that label as a filter.

Reply:
> With AllDisksArePVCs/VMSharedDisks we're already watching PVCs in this controller - and we're client.Get() PVCs elsewhere - so there isn't a reason anymore for a separate watcher for just the instance storage PCVs.

Yes, makes sense. 

> I don't think we are but if we're using that reader to do a List() then when using the regular client we should include that label as a filter.

No, we do not use directly. The PVCs get retrieved each individually by getting the names from the vm.spec.volumes